apps/condo/domains/user/schema/User.js

```diff
@@ -174,6 +174,12 @@ const User = new GQLListSchema('User', {
             kmigratorOptions: { null: true, unique: false },
         },
 
+        customAccess: {
```

Comment:
Is it possible to add values validation? From first look, we can validate list keys against existing schema names and set of CRUD operations and Boolean values.

Reply:
I think it is. But we need to pull all schema names only for validation purposes. Anyway this field couldn't be modified from admin panel